cmd/main.go

```diff
@@ -79,136 +56,34 @@ func main() {
 	if customEntriesPath != "" && customEntriesFormat == "" {
 		panic("error, variable customEntriesFormat is not set")
 	}
-
+	// 1.generate the endpointslice matrix
```

Comment:
I don't think you have to number the steps and the comments is also not needed here since its clear enough that commatrix.New creating a commantrix